hibernate-core/src/main/java/org/hibernate/engine/transaction/jta/platform/internal/TongWebJtaPlatform.java

```diff
@@ -0,0 +1,23 @@
+package org.hibernate.engine.transaction.jta.platform.internal;
```

Comment:
Please add the standard header that we use on all classes of the project.
Thanks!